.vib/consul/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/consul"
         },
-        "runtime_parameters": "Y29udGFpbmVyUG9ydHM6CiAgaHR0cDogODUwMApzZXJ2aWNlOgogIHBvcnRzOgogICAgaHR0cDogODAKICB0eXBlOiBMb2FkQmFsYW5jZXIKZGF0YWNlbnRlck5hbWU6IGRhdGFjZW50ZXIxCnJlcGxpY2FDb3VudDogMwo=",
+        "runtime_parameters": "c2VydmljZToKICBwb3J0czoKICAgIGh0dHA6IDgwCiAgdHlwZTogTG9hZEJhbGFuY2VyCmRhdGFjZW50ZXJOYW1lOiBkYXRhY2VudGVyMQpyZXBsaWNhQ291bnQ6IDMK",
```

Comment:
You shouldn't remove the `containerPorts.http` param, it is used by Goss. The Cypress envs below should be updated to the ones used in `cypress.json`